arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1399,7 +1396,6 @@ std::shared_ptr<LogicalCollection> ClusterInfo::getCollectionNT(DatabaseID const
   int tries = 0;
 
   if (!_planProt.isValid) {
-    loadPlan();
     ++tries;
```

Comment:
Should probably be replaced by a `waitForPlan(1);` The retry loop might also no longer make sense.